pkg/k8s/application/k8sApplicationService.go

```diff
@@ -454,6 +454,9 @@ func (impl *K8sApplicationServiceImpl) GetAllApiResourceGVKWithoutAuthorization(
 				k8sEventIndex = index
 			}
 		}
+		if gvk.Kind == "Node" {
+			allApiResources = append(allApiResources[:index], allApiResources[index+1:]...)
```

Comment:
what is the time complexity of this operation?

Reply:
it is linear but the scale pf objects is around 20